inference/core/models/roboflow.py

```diff
@@ -116,7 +117,13 @@ def __init__(
         self.metrics = {"num_inferences": 0, "avg_inference_time": 0.0}
         self.api_key = api_key if api_key else API_KEY
         model_id = resolve_roboflow_model_alias(model_id=model_id)
-        self.dataset_id, self.version_id = model_id.split("/")
+        # TODO:
+        #  Is this really all we had to do here?, think we don't even need it?
```

Comment:
To answer this question, it seems like the version ID is primarily used in models like SAM where there are multiple versions of the foundation model that can be called. Everywhere else, it's mainly used for providing clear debug info.